internal/services/loadbalancer/probe_resource.go

```diff
@@ -129,6 +136,9 @@ func resourceArmLoadBalancerProbeCreateUpdate(d *pluginsdk.ResourceData, meta in
 		}
 		return fmt.Errorf("failed to retrieve Load Balancer %q (resource group %q) for Probe %q: %+v", id.LoadBalancerName, id.ResourceGroup, id.ProbeName, err)
 	}
+	if d.IsNewResource() {
+		log.Printf("is new resource")
+	}
```

Comment:
Some logic missing here?

Reply:
@stephybun Thanks for reviewing. Just for testing. It has been removed.